CHANGELOG.md

```diff
@@ -13,12 +24,12 @@
 
 ### Changes
 
-* [BUGFIX]
-* [BUGFIX] Fix node_textfile_mtime_seconds to work properly on symlinks #1326
-* [CHANGE] Renamed `interface` label to `device` in netclass collector #1224
 * [BUGFIX] Add fallback for missing /proc/1/mounts #1172
+* [BUGFIX] Fix node_textfile_mtime_seconds to work properly on symlinks #1326
+* [CHANGE] Bonding state uses mii_status #1124
 * [CHANGE] Add TCPSynRetrans to netstat default filter #1143
```

Comment:
More an ENHANCEMENT, removing it would be breaking

Reply:
Agreed.